_episodes/007-zola.md

```diff
@@ -0,0 +1,110 @@
+---
+layout: post
+title: "Creating Static Sites in Rust with Vincent Prouillet"
+date: "2019-12-19:23:00+0000"
+file: https://audio.rustacean-station.org/file/rustacean-station/rustacean-station-e007-zola-vincent-prouillet.mp3
+duration: "54:24"
+length: "52224834"
+#reddit: (leave blank on initial publish, amend with link and uncomment this line after Reddit thread has been posted)
+
+# https://github.com/jekyll/jekyll/issues/7744
+layout: episode
+---
+
+[Vincent Prouillet](https://www.vincentprouillet.com/) talks about his experience building the Zola static site generator and reflects back on five years of working with Rust.
+
+Vincent is the creator of [Zola](https://www.getzola.org) (formerly known as Gutenberg), a static site generator built with [Rust](https://www.rust-lang.org/) and [Tera](https://github.com/Keats/tera), a Jinja2-like template engine.
```

Comment:
There's a bit of repetition between this paragraph and the previous one (they could probably be combined), and we probably don't need to link to the Rust website on this podcast, but don't consider these blockers.